src/components/Attachments/AttachmentCarousel/Pager/index.tsx

```diff
@@ -52,9 +53,11 @@ function AttachmentCarouselPager({items, renderItem, initialIndex, onPageSelecte
     }, []);
 
     useEffect(() => {
-        setActivePageState(initialIndex);
-        activePage.value = initialIndex;
-    }, [activePage, initialIndex]);
+        setActivePageState(initialPage);
+        activePage.value = initialPage;
+    }, [activePage, initialPage]);
+
+    const itemsMeta = useMemo(() => items.map((item, index) => ({source: item.source, index, isActive: index === activePageState})), [activePageState, items]);
```

Comment:
I am not sure if the naming is odd or I just don't understand what you mean by meta, so can you add a comment, please?

Reply:
renamed and added comments. You're right, the name was kinda weird